File: ijacampii-web/ijacampii-symfony/src/Entity/ParticipementEvent.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use JsonSerializable;

/**
 * ParticipementEvent
 *
 * @ORM\Table(name="participement_event", indexes={@ORM\Index(name="id_Event", columns={"id_Event"}), @ORM\Index(name="id_user", columns={"id_user"})})
 * @ORM\Entity
 */
class ParticipementEvent implements JsonSerializable
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_participement", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idParticipement;

    /**
     * @var int
     * @Assert\NotBlank(message="Type doit être non vide")
     * @Assert\NotNull
     * @Assert\Positive
     * @ORM\Column(name="nbr_participement", type="integer", nullable=false)
     */
    private $nbrParticipement;

    /**
     * @var \Utilisateurs
     *
     * @ORM\ManyToOne(targetEntity="Utilisateurs")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_user", referencedColumnName="id_user")
     * })
     */
    private $idUser;

    /**
     * @var \Evenement
     *
     * @ORM\ManyToOne(targetEntity="Evenement")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_Event", referencedColumnName="id_Event", onDelete="CASCADE")
     * })
     */
    private $idEvent;

    public function getIdParticipement(): ?int
    {
        return $this->idParticipement;
    }

    public function getNbrParticipement(): ?int
    {
        return $this->nbrParticipement;
    }

    public function setNbrParticipement(int $nbrParticipement): self
    {
        $this->nbrParticipement = $nbrParticipement;

        return $this;
    }

    public function getIdUser(): ?Utilisateurs
    {
        return $this->idUser;
    }

    public function setIdUser(?Utilisateurs $idUser): self
    {
        $this->idUser = $idUser;

        return $this;
    }

    public function getIdEvent(): ?Evenement
    {
        return $this->idEvent;
    }

    public function setIdEvent(?Evenement $idEvent): self
    {
        $this->idEvent = $idEvent;

        return $this;
    }
    public function jsonSerialize(): array
    {
        return array(
            'id' => $this->idParticipement,
            'utilisateurs' => $this->idUser,
            'nbrParticipement' => $this->nbrParticipement,
            'evenement' => $this->idEvent

        );
    }

    public function setUp($utilisateurs, $nbrParticipement, $evenement)
    {
        $this->idUser = $utilisateurs;
        $this->nbrParticipement = $nbrParticipement;
        $this->idEvent = $evenement;

    }

}

File: ijacampii-mobile/ijacampii/src/com/ijacampii/entities/ParticipementEvent.java
package com.ijacampii.entities;

public class ParticipementEvent {

    private int id;
    private Utilisateurs utilisateurs;
    private int nbrParticipement;
    private Evenement evenement;

    public ParticipementEvent() {
    }

    public ParticipementEvent(int id, Utilisateurs utilisateurs, int nbrParticipement, Evenement evenement) {
        this.id = id;
        this.utilisateurs = utilisateurs;
        this.nbrParticipement = nbrParticipement;
        this.evenement = evenement;
    }

    public ParticipementEvent(Utilisateurs utilisateurs, int nbrParticipement, Evenement evenement) {
        this.utilisateurs = utilisateurs;
        this.nbrParticipement = nbrParticipement;
        this.evenement = evenement;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Utilisateurs getUtilisateurs() {
        return utilisateurs;
    }

    public void setUtilisateurs(Utilisateurs utilisateurs) {
        this.utilisateurs = utilisateurs;
    }

    public int getNbrParticipement() {
        return nbrParticipement;
    }

    public void setNbrParticipement(int nbrParticipement) {
        this.nbrParticipement = nbrParticipement;
    }

    public Evenement getEvenement() {
        return evenement;
    }

    public void setEvenement(Evenement evenement) {
        this.evenement = evenement;
    }


}